src/rules/cyclomaticComplexityRule.ts

```diff
@@ -105,6 +105,7 @@ function walk(ctx: Lint.WalkContext<{ threshold: number }>): void {
 function increasesComplexity(node: ts.Node): boolean {
     switch (node.kind) {
         case ts.SyntaxKind.CaseClause:
+            return (node as ts.CaseClause).statements.length > 0;
```

Comment:
`if (foo === 0 || foo === 1) {bar()}` increases complexity twice. The will increase complexity only once with this change:
```ts
switch (foo) {
    case 0:
    case 1:
        bar();
}
```